presto-main/src/main/java/com/facebook/presto/sql/planner/ExpressionInterpreter.java

```diff
@@ -1156,6 +1160,17 @@ protected Object visitSubscriptExpression(SubscriptExpression node, Object conte
                 return new SubscriptExpression(toExpression(base, type(node.getBase())), toExpression(index, type(node.getIndex())));
             }
 
+            if (base instanceof SingleRowBlock) {
```

Comment:
nit: don't drop comments from https://github.com/prestosql/presto/pull/895/files#diff-bf0b73e8c2af7cb91f3dd7d82c70fa1aR1174  (and on line 1173 as well)